keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -457,23 +516,23 @@ The Topology Manager E2E test will enable the Topology Manager
 feature gate and set the CPU Manager policy to 'static'.
 
 At the beginning of the test, the code will determine if the system
-under test has support for single or multi-NUMA nodes. 
+under test has support for single or multi-NUMA nodes.
 
 ## Single NUMA Systems Tests
-For each of the four topology manager policies, the tests will
-run a subset of the current CPU Manager tests. This includes spinning 
-up non-guaranteed pods, guaranteed pods, and multiple guaranteed and 
-non-guaranteed pods. As with the CPU Manager tests, CPU assignment is 
-validated. Tests related to multi-NUMA systems will be skipped, and 
+For each of the five topology manager policies, the tests will
+run a subset of the current CPU Manager tests. This includes spinning
+up non-guaranteed pods, guaranteed pods, and multiple guaranteed and
+non-guaranteed pods. As with the CPU Manager tests, CPU assignment is
+validated. Tests related to multi-NUMA systems will be skipped, and
 a log will be generated indicating such.
 
 ## Multi-NUMA Systems Tests
-For each of the four topology manager policies, the tests will spin up
-guaranteed pods and non-guaranteed pods, requesting CPU and device 
-resources. When the policy is set to single-numa-node for guaranteed pods, 
-the test will verify that guaranteed pods resources (CPU and devices) 
-are aligned on the same NUMA node. Initially, the test will request 
-SR-IOV devices, utilizing the SR-IOV device plugin. 
+For each of the five topology manager policies, the tests will spin up
+guaranteed pods and non-guaranteed pods, requesting CPU and device
+resources. When the policy is set to single-numa-node for guaranteed pods,
+the test will verify that guaranteed pods resources (CPU and devices)
+are aligned on the same NUMA node. Initially, the test will request
+SR-IOV devices, utilizing the SR-IOV device plugin.
```

Comment:
Will we mention tests for `pod-level-single-numa-node`?

Reply:
I think we should update a tests for the changes that mentioned in this PR.
- add some test case for when dynamic policy is configured including `pod-level-single-numa-node` case.

I think maybe adding `update test` on ## Beta (v1.20) is sufficient.

Reply:
Done